.github/workflows/reusable/setup-notebook-testing/action.yml

```diff
@@ -0,0 +1,55 @@
+# This code is a Qiskit project.
```

Comment:
I think the convention is to put this in `.github/actions/` rather than `.github/workflows`

Reply:
Done, thanks!